android-radar/src/main/java/com/cedexis/androidradar/RadarWebView.java

```diff
@@ -73,10 +73,15 @@ public void start(final int zoneId, final int customerId, final RadarScheme sche
         } else if (webView == null) {
             throw new IllegalAccessError("Call Radar#init method before sending Radar events");
         } else {
-            configureWebView(webView);
+            WebSettings settings = webView.getSettings();
+            settings.setJavaScriptEnabled(true);
+            settings.setAllowFileAccess(true);
+            settings.setAllowContentAccess(true);
+            settings.setAllowFileAccessFromFileURLs(true);
+            settings.setAllowUniversalAccessFromFileURLs(true);
```

Comment:
This block appears to replace a block that did version checking before executing some of these commands. Are these safe to execute on older versions?

Reply:
There is a stronger check guarding this block than the old one, nevermind.